android/build.gradle

```diff
@@ -24,11 +24,10 @@ buildscript {
     repositories {
         google()
         mavenCentral()
-        jcenter()
     }
     dependencies {
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
-        classpath 'com.android.tools.build:gradle:4.1.0'
+        classpath 'com.android.tools.build:gradle:4.2.1'
```

Comment:
android build: Upgrade Android Gradle Plugin to v4.2.1
    
    Done to follow the template-app change in
    facebook/react-native@7599593b3, on the path to the RN v0.65
    upgrade.
    
    Release notes here:
      https://developer.android.com/studio/releases/gradle-plugin#4-2-0
    
    Quite a few changes announced; hopefully they're all good ones. Our
    tests pass, and a debug build on Android built and ran fine for me.

Cool. I'm looking through those release notes. This one is interesting:

> Starting in AGP 4.2, it is no longer possible to override Gradle properties from subprojects. In other words, if you declare a property in a gradle.properties file in a subproject instead of the root project, it will be ignored.

We have a fair number of such files supplied by different third-party libraries. You can see all of them (plus a few that don't apply, in template or example apps rather than live subprojects) like so:
```
$ find -name gradle.properties | xargs rg -v '^ *($|#)'
```

E.g.,
```
./node_modules/react-native-webview/android/gradle.properties
1:ReactNativeWebView_kotlinVersion=1.3.50
2:ReactNativeWebView_webkitVersion=1.4.0
3:ReactNativeWebView_compileSdkVersion=29
4:ReactNativeWebView_buildToolsVersion=29.0.3
5:ReactNativeWebView_targetSdkVersion=28
6:ReactNativeWebView_minSdkVersion=21
```

Glad to not have any of those applying ad hoc anymore!

I think that's the only one that affects us.